xml/System.Buffers.Binary/BinaryPrimitives.xml

```diff
@@ -80,10 +89,19 @@
         <Parameter Name="source" Type="System.ReadOnlySpan&lt;System.Byte&gt;" />
       </Parameters>
       <Docs>
-        <param name="source">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
-        <remarks>To be added.</remarks>
+        <param name="source">The read-only span to read.</param>
+        <summary>Reads an <see cref="T:System.Int16" /> out of a read-only span of bytes as little endian.</summary>
+        <returns>The little endian value.</returns>
+        <remarks>
+          <format type="text/markdown"><![CDATA[
+
+## Remarks
```

Comment:
It's unfortunate that we have to put this remarks heading each time. Seems redundant, since we are already stating this is the remarks tag. Maybe, the process to convert this xml to a format that is rendered on the site should understand this and format appropriately (even if the value of the tag is formatted as markdown).

Reply:
It's not required but the way this works is that the examples section can also go in here, so this way is ready to add that when needed. In the future, everything will be in Markdown, then it'll probably be easier.